mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -1736,6 +1736,41 @@ pub async fn enable_bch_with_tokens(
     json::from_str(&enable.1).unwrap()
 }
 
+pub async fn enable_bch_with_tokens_without_balance(
```

Comment:
I placed it under `enable_bch_with_tokens` so it can be used in `bch_and_slp_tests` if needed.  https://github.com/KomodoPlatform/atomicDEX-API/blob/c5e8fe1cd2a2450813d732010d56b7e9405c9fab/mm2src/mm2_test_helpers/src/for_tests.rs#L1705 `mm2_test_helpers` is a common crate for test functions/structs/helpers, maybe I should move `enable_eth_with_tokens_without_balance` and `enable_eth_with_tokens` there instead. What do you think? The idea is that functions that wrap requests should be accessible in tests anywhere if needed.

Reply:
enable_bch_with_tokens_without_balance -> slp_tests.rs
enable_tendermint_without_balance -> tendermint_tests.rs

Reply:
I guess this won't cause a problem, will do it. But it might be moved again to `mm2_test_helpers` in the future if a test in another crate requires it :)

Reply:
done

Reply:
It also would make a code much more clean if you put `enable_bch_with_tokens_without_balance` right after the only place where it's called like that:

![image](https://user-images.githubusercontent.com/801971/235074820-6e54abbe-7fba-4da8-ad0d-fc77042ec48c.png)

Reply:
> It also would make a code much more clean if you put enable_bch_with_tokens_without_balance right after the only place where it's called like that:

It's considered good practice for test utility functions such as `enable_bch_with_tokens_without_balance` to be put at the beginning of the test module before the actual unit test functions, this is also the convention followed through out the project. This improves code readability where you see all the functions that can be reused at the start of the file.

Reply:
I would not suggest if it's used somewhere else in fact and then would not be considered as part of a certain test

Reply:
It can easily be used in other tests in the future since this function's cost is lower with no calls for balances if they are not needed, that's why a clear structure from the start is better. Also, if you check other popular rust repos and popular rust books examples, they all do it this way.
An alternative idiomatic way is to have the function nested inside the unit test if it's used only once, but for rpc calls, from past expereince can see that they get used multiple times later.